inst/stan/wwinference.stan

```diff
@@ -117,7 +128,7 @@ parameters {
   real<lower=0> sigma_rt; // magnitude of site level variation from state level
   real<lower=0, upper=1> autoreg_rt_site;
   real<lower=0, upper=1> autoreg_p_hosp;
-  matrix[n_subpops, n_weeks] error_site; // matrix of subpopulations
+  ////matrix[n_subpops, n_weeks] error_site; // matrix of subpopulations
```

Comment:
Was going to suggest getting rid of this, and I think you should get rid of any "commented" code. But if it will help you make the modularity switch then I think its fine to leave it for now (though I think we would do that with the infrastucture you've written but passing in an identity matrix) instead of `sigma_mat`